modules/table/src/main/java/org/apache/ignite/internal/table/distributed/index/IndexMetaStorage.java

```diff
@@ -154,78 +169,86 @@ Collection<IndexMeta> indexMetasSnapshot() {
         return List.copyOf(indexMetaByIndexId.values());
     }
 
-    private void onCatalogIndexCreateEvent(CreateIndexEventParameters parameters) {
+    private CompletableFuture<Boolean> onCatalogIndexCreateEvent(CreateIndexEventParameters parameters) {
```

Comment:
```suggestion
    private CompletableFuture<Boolean> updateMetastoreOnCatalogIndexCreateEvent(CreateIndexEventParameters parameters) {
```

Let's change the name (or add doc?) to convey what happens on the event**s** and what does boolean in the future mean

Reply:
The name may not be entirely obvious at first glance, but it has a certain pattern, and is found in our project:
`on...Event` - says that this is a reaction to an event.
`Catalog` - indicates that this is a `CatalogEvent`.
`IndexCreate` - a specific event when an index is created in a catalog.

I don't see any need for documentation since the described name and argument pattern self-documents the method.